Frontend/src/components/meeting/MeetingModal.jsx

```diff
@@ -14,35 +14,57 @@ function MeetingModal({
   declineMeeting,
   selectedMeeting,
 }) {
-  // const formattedStartTime = moment(selectedMeeting.scheduledTime).format('MMMM Do YYYY, h:mm:ss a');
-  // const formattedEndTime = moment(selectedMeeting.start).add(30, 'minutes').format('MMMM Do YYYY, h:mm:ss a');
+  const formattedDate = moment(selectedMeeting.end).format("MMMM Do YYYY");
+  const formattedStartTime = moment(selectedMeeting.start).format("h:mm a");
+  const formattedEndTime = moment(selectedMeeting.end).format("h:mm a");
 
   return (
     <>
-      <div className="modal" id="match">
-        <div className="modal-content">
-          <span className="modal-close" onClick={toggleModal}>
-            ×
-          </span>
-          <div className="mm-body">
-            <h3>{selectedMeeting.title}</h3>
-            <p>Status: {selectedMeeting.status}</p>
-            <p>Mentor: {selectedMeeting.mentorName}</p>
-            <p>Attendees:</p>
+      <Modal open={true} onClose={toggleModal}>
+        <Box
+          sx={{
```

Comment:
lets give this a better name

Reply:
What exactly should I name better? The modal, box, or something else?